website/src/styles/settings.scss

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
is this effectively an empty file?

Reply:
Yes, that's part of the template provided by vuetify